### PR TITLE
readability improvements communication, data-structures, function-structures, installation & operation modes

### PR DESCRIPTION
**Pull request by aedler-ft**

### CODE REVIEW

File: doc/appendix-at-rksv/communication/communication.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Communication
 
-This chapter expands on the descriptions of the ways to communicate with the fiskaltrust.Middleware covered in the Chapter ["Communication"](../../general/communication/communication.md) of the General Part, with country specific information applicable to the Austrian market.
+This chapter expands on the descriptions of the ways to communicate with the fiskaltrust.Middleware covered in the Chapter ["Communication"](../../general/communication/communication.md) of the General Part, with country-specific information applicable to the Austrian market.
```
Comment (DoxaPap) [inline doc/appendix-at-rksv/communication/communication.md:3]: this chapter expands more on describing the ways to communicate..

File: doc/appendix-at-rksv/data-structures/data-structures.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Data Structures
 
-This chapter expands on the descriptions of the data structures covered in the Chapter ["Data Structures"](../../general/data-structures/data-structures.md) of the General Part, with country specific information applicable to the Austrian market.
+This chapter expands on the descriptions of the data structures covered in the Chapter ["Data Structures"](../../general/data-structures/data-structures.md) of the General Part, with country-specific information applicable to the Austrian market.
```
Comment (DoxaPap) [inline doc/appendix-at-rksv/data-structures/data-structures.md:3]: this chapter expands more on describing the data structures..

File: doc/appendix-at-rksv/function-structures/function-structures.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Function Structures
 
-This chapter expands on the descriptions of the function structures covered in the Chapter ["Function structures"](../../general/function-structures/function-structures.md) of the General Part, with country specific information applicable to the Austrian market.
+This chapter expands on the descriptions of the function structures covered in the Chapter ["Function structures"](../../general/function-structures/function-structures.md) of the General Part, with country-specific information applicable to the Austrian market.
```
Comment (DoxaPap) [inline doc/appendix-at-rksv/function-structures/function-structures.md:3]: More detailed descriptions of the function structures covered in Chapter bla can be found here
(so it is not monotonous)

File: doc/appendix-at-rksv/function-structures/function-structures.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ The CurrentDateTime is provided in the `yyyyMMddhhmmssfff` format.
 
 ### IATSSCD Interface
 
-This interface is applicable only for the Austrian market and enables direct communication with the signature creation device for own purposes: it can be used for testing if the fiskaltrust.Middleware is running ("Echo" call), for getting the certificate ("Certificate" call), or signing autonomously ("Sign" call).
+This interface is applicable only for the Austrian market and enables direct communication with the signature creation device for own purposes: it can be used for testing if the fiskaltrust.Middleware is running ("Echo" call), for requesting the certificate ("Certificate" call), or signing autonomously ("Sign" call).
```
Comment (DoxaPap) [inline doc/appendix-at-rksv/function-structures/function-structures.md:31]: or for signing autonomously

File: doc/appendix-at-rksv/communication/communication.md (hunk 1 above)
Comment (aedler-ft) [reply doc/appendix-at-rksv/communication/communication.md]: brilliant!

File: doc/appendix-at-rksv/data-structures/data-structures.md (hunk 2 above)
Comment (aedler-ft) [reply doc/appendix-at-rksv/data-structures/data-structures.md]: done

File: doc/appendix-at-rksv/function-structures/function-structures.md (hunk 3 above)
Comment (aedler-ft) [reply doc/appendix-at-rksv/function-structures/function-structures.md]: ;-)

**Review by TSchmiedlechner — APPROVED** [review]

**Pull request merged**